modules/ROOT/pages/custom-menu-toolbar-button.adoc

```diff
@@ -12,10 +12,11 @@ For example: The table plugin's `+table+` toolbar button opens a menu similar to
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|fetch |`+(success: (menu) => void) => void+` |required |Function that takes a callback which must be passed the list of options for the button's dropdown.
+|fetch |`+(success: (menu) => void, fetchContext) => void+` |required |Function that takes a callback and a `+fetchContext+` object. The callback function must be passed the list of options for the button's dropdown. The `+fetchContext+` object provides information which is useful for choosing which items should be passed to the callback. For more details on `fetchContext`, see xref:searchable-menu-buttons[Searchable Menu Buttons]
```

Comment:
I looked in various places, and we didn't seem to have a standard way of specifying the types of objects passed into functions, so because that detail is necessary to use this function, I've just put a small amount of detail in the table, and then linked to a lower section that has more information. The lower section still doesn't explicitly state the types, though. As I said, other examples (https://www.staging.tiny.cloud/docs/tinymce/6/dialog-configuration/) aren't particularly consistent on how they detail the information in objects passed through to functions. In some cases, the information doesn't seem to be present at all.

The admonition about this being only in >= 6.2 was also moved to the linked section, instead of putting even more content in this table (as was discussed).

Reply:
Since this option is used in all menu buttons, not just the searchable menu button, I don't think referencing the searchable menu buttons makes this clear. I wonder if stating that there are examples of this function in both live demos.

Reply:
I wonder if the `Searchable Menu Buttons` part should just be called something closer to `FetchContext`. Let's clear this up with the team. This page differs from some of the others, which has made it hard to apply precedent and consistency.

Reply:
We'll address this later as part of a "Cleanup Searchable Menu docs page"